paymentsheet/res/drawable-night/stripe_ic_paymentsheet_pm_affirm.xml

```diff
@@ -0,0 +1,12 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Were you able to get the tint of this icon correct?

Reply:
I think so - I just set the fill color to `"#FFFFFF"`